doc/engineering-reference/src/simulation-models-encyclopedic-reference-001/coils.tex

```diff
@@ -1611,38 +1611,38 @@ \subsubsection{Special Calculations for Coil:Cooling:DX:TwoStageWithHumidityCont
 
 1. No latent control
 
-   Perform normal operation with SHR_{Normal}
+   Perform normal operation with SHR\(_{Normal}\)
```

Comment:
The output of these changes look identical on my machine. What's the purpose?

Reply:
Another warning and, again, the LaTeX compiler is able to guess in this instance and muddle through. What's wrong is the `_{blah}` stuff is not valid outside of math mode. The `\(` and `\)` puts us in math mode (similar to `$`...`$` but allows checking for balancing).